### ISSUE TITLE
Feature request: per credential setting to skip login prompt

### ISSUE THREAD

**Issue by simtomas** [limpkin/mooltipass#586]
### Missing feature

Setting (per credential) option to skip Login prompt and go directly to password
(as it is currently for empty Logins).

### Justification
Some credentials don't usually require a Login (just password and optionally TOTP),
for example SSH logins.
However it is convenient to group them in the mooltipass as follows:
Service: SSH
Login: server_name
Password: password

So Login in used to select the entry from the list, but is not used in the actual login process.

### Workarounds
One could skip using grouping, so the above example would be
Service: SSH_server_name
Login: &lt;empty&gt;
Password: password

But given many servers, this would pollute top level list of services in mooltipass.

There is also option to turn wheel from V to X on login prompt - which I use.

In short - this feature would save one mooltipass prompt, skipping "Type Login?" prompt all together for selected credentials.

**Comment by limpkin** [comment]
hello!
this is the wrong repository... could you file that feature request in https://github.com/mooltipass/minible ?
